server/mergin/sync/schemas.py

```diff
@@ -57,7 +60,7 @@ def insert_usernames(self, data, **kwargs):
 
 def project_user_permissions(project):
     return {
-        "upload": ProjectPermissions.Upload.check(project, current_user),
+        "upload": ProjectPermissions.Edit.check(project, current_user),
```

Comment:
This mapping is used by mobile and mergin client to check if it is possible to make push to server. We can rename it in future upload -> Edit and add new Write key.

Reply:
As I mentioned in PR description, there is inconsistency between "upload" and push permissions in case, that user is editor and do something not in editors rule defined (conflit, adding qgs file, ...).

Reply:
it is just good to have in in code so we don't forget :-)